src/prefect/utilities/context.py

```diff
@@ -57,8 +57,8 @@
 import threading
 from typing import Any, Iterator, MutableMapping
 
-from prefect.configuration import config
-from prefect.utilities.collections import DotDict
+from prefect.configuration import config, Config
```

Comment:
```suggestion
from prefect.configuration import config
```

Reply:
I need this class when calling `as_nested_dict` in like 82

Reply:
🤦‍♂ sorry